channel.js

```diff
@@ -0,0 +1,23 @@
+class Channel {
+    constructor(id, guild, name, type) {
+        this.id = id;
+        this.guild = guild;
+        this.name = name;
+        this.type = type;
+       
+    }
+    getChannelData(){

```

Comment:
we could use the instance of the class directly instead of creating an extra function for returning the object